Dockerfile

```diff
@@ -3,6 +3,10 @@ MAINTAINER Johannes Mitlmeier <dev.jojomi@yahoo.com>
 
 COPY ./run.sh /run.sh
 ENV HUGO_VERSION=0.32.2
+
+#Pick a suitably high GID for sudo group. 
+RUN addgroup sudo -g 4001 
```

Comment:
Instead of calling the user `sudo`, could it be named `hugo`? It would help avoid confusion with the `sudo` command.

Reply:
Do you mean, the group - hugo? Yes, of course. So we'll add the group hugo to valid sudoers.

Reply:
Oh, sorry, yeah I meant group, and not user. Naming the group `hugo` would help avoid ambiguity.

A copy and paste of what I did in a fork of `docker-hugo`:

```
# Add a `hugo` user.
#
# Docker's _Security_ guide has a [Conclusions](https://docs.docker.com/engine/security/security/#conclusions) section that recommends, for added security, configuring a non-root user to run commands.
#
# With a `hugo` user configured the `hugo` user can be accessed by passing `--user hugo` to `docker run`.
#
# Some background on Docker security best practices - https://groups.google.com/forum/#!msg/docker-user/e9RkC4y-21E/JOZF8H-PfYsJ
RUN addgroup hugo
RUN adduser -G hugo -s /bin/sh -D hugo
```

As you can see, I added a group named `hugo`, and then added a user named `hugo`, setting the user's group to `hugo`.

I then switched to the new user and group before running the `hugo` command itself:

```
# Switch to the `hugo` user for all subsequent commands. Configuring the `USER` also causes all commands executed by `docker run` to run as the `hugo` user by default.
USER hugo:hugo
```